### ISSUE TITLE
Alacritty only update buffer if a new instance of alacritty is closed/opened.

### ISSUE THREAD

**Issue by MaKaNu** [alacritty/alacritty#6643]
For bug reports, the following information can help speed up the process. Please
describe the bug that you have found and what you would expect to happen
instead.

### System

OS: Linux
Version: 0.11.0
Linux/arch: X11 LEFTWM picom

### Logs

Crashes: STDERR, STDOUT
Font/Terminal size: monospace

I have a fresh installation of arch with leftwm as my WM installed. I use the integrated GPU Intel Iris Xe Graphics. Every time I launch a new terminal it opens as expected and shows me the prompt. But any other action seems to acknowledged but not printed. 

An Example:

The Terminal prompt looks like this

```bash
[user@pc ~]$
```
I now enter a command like `ls` and the prompt still looks like this:

```bash
[user@pc ~]$
```
Next I open another instance of alacritty with "SUPER+SHIFT+ENTER". Now both Terminals are open. The old shows now the following:

```bash
[user@pc ~]$ ls
Downloads
[user@pc ~]
```
while the new one shows now only the prompt. If I want to enter a new command into the old terminal AND want to see what I am doing I need to repeat the steps from before. Since my the system is still mostly unconfigured I am unable to extract ouutput from `alacritty --print-events` but as far as I noticed I only found `INFO` on the output not a single `WARN`or `ERR`.

**Comment by chrisduerr** [comment]
You're using a tiling window manager, I'd assume what is actually causing the redraw is the window being resized. Can you confirm that resizing causes things to redraw?

Also please give me some hardware information, specifically GPU and graphics drivers (kernel and userspace ideally).

**Comment by kchibisov** [comment]
Might be fixed on master due to removal of vsync.

**Comment by MaKaNu** [comment]
In the meantime I was able to establish an intermediate setup, so I can provide more information now.

Some of my outputs:

**PCI devices:**
```
$ lspci -v
.
.
.
0000:00:02.0 VGA compatible controller: Intel Corporation Alder Lake-UP3 GT2 [Iris Xe Graphics] (rev 0c) (prog-if 00 [VGA controller])
	Subsystem: Dell Device 0b0c
	Flags: bus master, fast devsel, latency 0, IRQ 145, IOMMU group 0
	Memory at 6054000000 (64-bit, non-prefetchable) [size=16M]
	Memory at 4000000000 (64-bit, prefetchable) [size=256M]
	I/O ports at 3000 [size=64]
	Expansion ROM at 000c0000 [virtual] [disabled] [size=128K]
	Capabilities: <access denied>
	Kernel driver in use: i915
	Kernel modules: i915
.
.
.
```
**Driver:**

```
$ pacman -Qs intel
local/intel-ucode 20221108-1
    Microcode update files for Intel CPUs
local/libmfx 23.1.0-1
    Intel Media SDK dispatcher library
local/vulkan-intel 22.3.3-3
    Intel's Vulkan mesa driver
local/xf86-video-intel 1:2.99.917+916+g31486f40-2 (xorg-drivers)
    X.org Intel i810/i830/i915/945G/G965+ video drivers
```

```
$ pacman -Qs mesa
local/mesa 22.3.3-3
    An open-source implementation of the OpenGL specification
local/mesa-utils 8.5.0-2
    Essential Mesa utilities
local/vulkan-intel 22.3.3-3
    Intel's Vulkan mesa driver
```

And I also checked if resizing triggers the redraw, which I can confirm. 

Furthermore I had similar issue with other accelerated software: Kitty, Obsidian. Obsidian worked fine after I was successful to disable acceleration. Kitty showed the same redraw issues as alacritty. I have a gut feeling it might be the xf86-video-intel, but I had issue to sddm without it.

**Comment by chrisduerr** [comment]
Yeah Intel's graphics drivers are pretty bad. There's a good chance master fixes it so I recommend testing that, but if it doesn't work you're out of luck.

**Comment by MaKaNu** [comment]
What do you mean by master? The master branch? Since I have installed via pacman and not builded myself I thought I would be running on master branch. Or do you talking about something else?

**Comment by kchibisov** [comment]
pacman provides the latest release. you can install `-git` version from the aur.

**Comment by MaKaNu** [comment]
My problem got solved by uninstalling the intel driver and using modeset instead. Still have Vulcan and Mesa installed but removed `xf86-video-intel` like #6103 also mentions. After checking the issue I realized that I have overwritten the arch wiki note for 4th Gen and newer GPUs. Somehow wasn't able find #6103 before and as always had reading issues in the wiki. Sorry for bothering alacritty, but you guys are the first after a fresh install.
